File: src/saomiguelbus/lib/models/stop.dart
import 'index.dart';

class Stop {
  String name;
  Location location;

  Stop(this.name, this.location) {
    // if (!Datasource().getStops().contains(this)) {
    //   Datasource().addStop(this);
    // }
  }

  @override
  String toString() {
    return "$name, loc: $location";
  }

  Stop.fromJson(Map<String, dynamic> json)
      : name = json['name'],
        location = Location.fromJson(json['location']);

  Map<String, dynamic> toJson() => {
        'name': name,
        'location': location.toJson(),
      };
}

File: legacy/web/data/entities/stop.py
class Stop:
    def __init__(self, name, coordinates):
        self.name = name
        self.coordinates = coordinates


'''
class Stop constructor(var name: String, val coordinates: Location) {

    init {
        if (!Datasource().getStops().contains(this)) Datasource().addStop(this)
    }
    override fun toString(): String{
        return name
    }
}
'''